package.json

```diff
@@ -19,7 +26,13 @@
     "clean": "rm -rf ./dist/",
     "build:travisci": "ng build --target production --aot --build-optimizer --vendor-chunk --extract-css --output-hashing all --deploy-url /firecloud-explorer/ --base-href /firecloud-explorer",
     "ci": "npm run clean && ng lint && ng test --browser ChromeTravisCi --single-run --reporters spec --sourcemaps=false && npm run build:travisci",
-    "electron": "electron ."
+    "electron": "electron .",
+    "package-linux": "electron-packager . --overwrite --asar --platform=linux --arch=x64 --icon=src/assets/icons/png/1024x1024.png --prune=true --out=linux/builds",
+    "create-debian-installer": "electron-installer-debian --src 'linux/builds/FireCloud Explorer-linux-x64/' --arch amd64 --config debian.json",
+    "package-mac": "electron-packager . --overwrite --platform=darwin --arch=x64 --icon=src/assets/icons/mac/FCExplorer.icns --prune=true --out=mac/builds --app-bundle-id=org.broadinstitute.firecloud-explorer --app-copyright='Copyright © 2018 Broad Institute All Rights Reserved' --app-version='0.1.0' --app-category-type=public.app-category.medical",
```

Comment:
Are these commands supported? They appear to duplicate the ones in the script above. If they are duplicates, it should be documented (or, preferably, the duplicate one should be removed). If only one works, the non-working commands should be documented or removed for now.